tests/test_client.py

```diff
@@ -38,6 +38,76 @@ def test_druid_returns_error(self, mock_urlopen):
                     threshold=1,
                     context={"timeout": 1000})
 
+    @patch('pydruid.client.urllib.request.urlopen')
+    def test_druid_returns_html_error(self, mock_urlopen):
+        # given
+        message = """<html>
```

Comment:
NIT: there's this trick indent long strings in code but not in the variable itself:
```python
    s = textwrap.dedent"""\
    this
        wont
    be indented
    """
```